OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -67,6 +67,35 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
 						break;
 					}
 
+				case "ReplacePlayerWithBot":
+					{
+						var client = orderManager.LobbyInfo.ClientWithIndex(clientId);
+						if (client != null && world != null)
+						{
+							var player = world.FindPlayerByClient(client);
+							if (player == null || player.IsBot)
+								break;
+
+							client.State = Session.ClientState.Ready;
+
+							var admin = orderManager.LobbyInfo.Clients.FirstOrDefault(c => c.IsAdmin);
```

Comment:
The pr does not work as advertised:

> Added an option to replace disconnected players with AI bots

Players will not be replaced with an AI bot if they are the server admin.